cranelift/filetests/filetests/vcode/aarch64/bitops.clif

```diff
@@ -1,6 +1,34 @@
 test vcode
 target aarch64
 
+function %a(i8) -> i8 {
+block0(v0: i8):
+    v1 = bitrev v0
+    return v1
+}
+
+; check: stp fp, lr, [sp, #-16]!
+; nextln: mov fp, sp
+; nextln: rbit w0, w0
```

Comment:
For what it's worth, I think there's a way to start catching from this line, and stopping at the line that does the `lsr`. Unless you explicitly wanted to test that there was no stack space allocated and no other instructions generated for this? (which is nice for completeness)

Reply:
This is something I've wanted to clean up in our vcode filetests -- there should be a better way than matching a whole golden output. The reason for this for now is to ensure nothing else is generated, yes; I'll create an issue to track this desired improvement though.